demography.md

```diff
@@ -71,40 +93,61 @@ To fix this, let's add some migration events to the specific demographic history
 ## Migrations
 
 With msprime, you can specify continual rates of migrations between
-populations, as well as one-off mass migrations.
+populations, as well as admixture events, divergences and one-off mass migrations.
 
 ### Constant migration
 
 ![constant-migration-diagram](_static/tute-population-structure-1.png)
 
-Migration rates between the populations can be specified as the elements of an
-*N* by *N* numpy array, and given to {func}`msprime.sim_ancestry` via the
-``migration_matrix`` argument. The diagonal elements of this array must each be
-0, and the *(i, j)* th element specifies the expected number of migrants moving
-from population *j* to population *i* per generation, divided by the size of
-population *i*.  When this rate is small (close to 0), it is approximately
-equal to the fraction of population *i* that consists of new migrants from
-population *j* in each generation.
+Migration rates between the populations can be thought as the elements of an
+*N* by *N* numpy array, and are passed to our {class}`msprime.Demography` object individually via the {meth}`msprime.Demography.set_migration_rate`
+method.
```

Comment:
Worth mentioning the ``set_symmetric_migration_rate``, which saves a bit of typing when you have 2 or more populations exchaning migrants at the same rates.